fuzz/clamav_scanfile_fuzzer.cpp

```diff
@@ -119,15 +127,16 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
 {
     FILE* fuzzfile = NULL;
 
-    fuzzfile = fopen(kClamAVState.tmp_file_name, "w");
+    fuzzfile = fopen(kClamAVState.tmp_file_name.c_str(), "w");
     fwrite(data, size, 1, fuzzfile);
+    fflush(fuzzfile);
```

Comment:
Shouldn't be necessary.  `fclose()` is documented to do this, or the OS is broken.

Reply:
Oops, didn't mean to commit that.  Yeah it's not required.